test/units/cli/test_galaxy.py

```diff
@@ -1102,13 +1102,8 @@ def test_parse_requirements_in_older_format_illegal(requirements_cli, requiremen
 collections:
 - version: 1.0.0
 """], indirect=True)
-def test_parse_requirements_without_mandatory_name_key(requirements_cli, requirements_file):
-    # Used to be "Collections requirement entry should contain the key name."
-    # Should we check that either source or name is provided before using the dep resolver?
-
-    expected = "Neither the collection requirement entry key 'name', nor 'source' point to a concrete resolvable collection artifact. "
-    expected += r"Also 'name' is not an FQCN\. A valid collection name must be in the format <namespace>\.<collection>\. "
-    expected += r"Please make sure that the namespace and the collection name contain characters from \[a\-zA\-Z0\-9_\] only\."
+def test_parse_requirements_contains_name_key(requirements_cli, requirements_file):
+    expected = "Failed to parse collection requirement entry: {'version': '1.0.0'}: missing required arguments: name"
```

Comment:
The error message with a phrase separated with 3 colons and has another colon in a piece of structured data looks weird to me. It's probably worth rephrasing.